src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
+Animal.removeKilled = function() {
+  Animal.alive = Animal.alive.filter(animal => animal.health > 0);
+};
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
+    super(name, health);
+    this.hidden = hidden;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
```

Comment:
According to the checklist: _if the function has a name hide this function should just hide but not toggle_
```suggestion
    this.hidden = true;
```

Reply:
Sure, I've forgotten this point of the checklist.